srunner/scenarioconfigs/osc2_scenario_configuration.py

```diff
@@ -17,11 +17,16 @@
 from srunner.osc2_dm.physical_object import *
 from srunner.osc2_dm.physical_types import Physical, Range
 
-# 标准库
 from srunner.osc2_stdlib.path import Path
 
+<<<<<<< HEAD
```

Comment:
Pretty sure this isn't supposed to be merged :)

Reply:
Fixed, thanks for noticing it.